nf_core/pipelines/create/custompipeline.py

```diff
@@ -51,6 +51,13 @@
 - prettier (https://github.com/prettier/prettier): enforces a consistent style (indentation, quoting, line length, etc).
 """
 
+markdown_citations = """
+If adding citations, the pipeline template will contain a `CITATIONS.md` file to add the citations of all tools used in the pipeline.
+
+Additionally, it will include a YAML file (`assets/methods_description_template.yml`) to describe a Materials & Methods section describing the tools used in the pieline,
```

Comment:
```suggestion
Additionally, it will include a YAML file (`assets/methods_description_template.yml`) to add a Materials & Methods section describing the tools used in the pieline,
```